src/tcache.c

```diff
@@ -116,9 +122,11 @@ tcache_bin_flush_small(tsd_t *tsd, tcache_t *tcache, tcache_bin_t *tbin,
 		item_extent[i] = iealloc(tsd_tsdn(tsd), *(tbin->avail - 1 - i));
 	}
 
+	bool merged_stats = false;
+	unsigned nflushed = 0, nskipped = 0;
```

Comment:
Style nit: For declarations incorporating initialization, only declare one variable per statement.